xtask/src/fpga.rs

```diff
@@ -253,18 +258,38 @@ pub(crate) fn fpga_entry(args: &Fpga) -> Result<()> {
             // Need to clone caliptra-mcu-sw to run tests.
             run_command(target_host.as_deref(), "[ -d caliptra-mcu-sw ] || git clone https://github.com/chipsalliance/caliptra-mcu-sw --branch=main --depth=1").expect("failed to clone caliptra-mcu-sw repo");
         }
-        Fpga::Test { target_host } => {
+        Fpga::Test {
+            target_host,
+            test_package,
+            test,
+        } => {
             println!("Running test suite on FPGA");
             is_module_loaded("io_module", target_host.as_deref())?;
             // Clear old test logs
             run_command(target_host.as_deref(), "(sudo rm /tmp/junit.xml || true)")?;
             // Run caliptra-mcu-sw test suite.
             // Ignore error so we still copy the logs.
-            let _ = run_command(target_host.as_deref(), "(cd caliptra-mcu-sw && \
+            let mut test_command = "(cd caliptra-mcu-sw && \
                 sudo CPTRA_FIRMWARE_BUNDLE=\"${HOME}/all-fw.zip\" \
                 cargo-nextest nextest run \
                 --workspace-remap=. --archive-file $HOME/caliptra-test-binaries.tar.zst \
-                -E \"package(mcu-hw-model) - test(model_emulated::test::test_new_unbooted)\" --test-threads=1 --no-fail-fast --profile=nightly)");
+                --test-threads=1 --no-fail-fast --profile=nightly "
+                .to_string();
+            if test_package.is_some() {
```

Comment:
WDYT of having a filter argument instead of package and test, so we can filter any way we want? Essentially it would be the same as the `-E` argument in nextest

Reply:
ah good point. Fixed. LMK what you guys think.